### PR TITLE
Markdown migration instructions

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: docs/software/markdown-migration.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+1. Create a branch based off of master
+2. Add page to [mkdocs.yml](../../mkdocs.yml) 
+3. Commit your changes and push to your GH repo
+4. Make PR containing the following tasks in the body body:
+
+        - [ ] Remaining TWiki link #1
+        - ...
+        - [ ] Remainng TWiki link #N
+        - [ ] Add migration header to TWiki document
```
Comment (bbockelm) [inline docs/software/markdown-migration.md]: Can you link to an example PR for the documentation project?

File: docs/software/markdown-migration.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+|:------------------------------------------------------|:-------------------|
+| `<span class="twiki-macro UCL\_PROMPT\_ROOT"></span>` | `[root@client ~]$` |
+| `<span class="twiki-macro UCL\_PROMPT"></span>`       | `[user@client ~]$` |
+
+#### Highlighting user input  ####
+
+Within command blocks and file snippets, we've used `%RED%...%ENDCOLOR%`, `&lt;...&gt;`, etc. to highlight areas that users would have to insert text specific to their site. For now, use desciptive, all-caps text wrapped in angle brackets to indicate user input:
+
+```
+<USER INPUT>
```
Comment (bbockelm) [inline docs/software/markdown-migration.md]: Please give an example?

File: docs/software/markdown-migration.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+This is an archive, find the new version of this document [[LINK TO GITHUB DOCUMENT][here]].
+
+Background:
+
+At the end of year (2017), the TWiki will be retired in favor of !GitHub. You can find the various TWiki webs and their new !GitHub locations listed below:
+
+   * Release3: https://opensciencegrid.github.io/docs
+   * !SoftwareTeam: https://opensciencegrid.github.io/technology
+</div>
+```
```
Comment (bbockelm) [inline docs/software/markdown-migration.md]: This section should contain an example of how to update backlinks of pages still in the twiki to point at the new GitHub document.


For example, the backlinks for https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SHA2Compliance are listed here: https://twiki.opensciencegrid.org/bin/oops/Documentation/Release3/SHA2Compliance?template=backlinksweb .  Those should be converted after PR has been merged.

File: docs/software/markdown-migration.md (hunk 1 above)
Comment (brianhlin) [inline docs/software/markdown-migration.md]: We haven't done any yet, can we add this afterwards?

File: docs/software/markdown-migration.md (hunk 3 above)
Comment (brianhlin) [inline docs/software/markdown-migration.md]: If we're replacing the entirety of the TWiki document's contents, what links would there be to update?

Comment (bbockelm) [inline docs/software/markdown-migration.md]: The backlinks are _other twiki pages_ that reference the one you are editing.

I.e., once SHA2Compliance is moved over, InstallOSGBestmanSE should be updated to link to GitHub.

File: docs/software/markdown-migration.md (hunk 1 above)
Comment (bbockelm) [inline docs/software/markdown-migration.md]: Sure, no problem.  I'll try one as soon as this gets merged.

File: docs/software/markdown-migration.md (hunk 3 above)
Comment (brianhlin) [inline docs/software/markdown-migration.md]: I can go either way on this but I think there's some value in taking the user to the old page that has the migration header:

1. They're informed of the GitHub transition
2. It's less jarring for the user

Comment (bbockelm) [inline docs/software/markdown-migration.md]: Sure.  I will make a note of this in the text and we will revisit later if needed.  I just didn't want to make it ambiguous for the reader.

**Comment by bbockelm** [comment]
If you have nothing else to change, please go ahead and merge.

**Pull request merged**